### PR TITLE
AB#39080 add app permissions sample query: GET teams channels

### PR DESCRIPTION
**Pull request by acchiang**
Copied from the other GET teams channels sample query, but with new generated GUID and added information in the tip to ensure the user knows to switch to user mode in order to get Teams IDs.

### CODE REVIEW

**Review by omersayshi — APPROVED** [review]

File: sample-queries/sample-queries.json [hunk 1]
```diff
@@ -2984,44 +2984,54 @@
...
+        "method": "GET",
+        "humanName": "get list of members in team",
+        "requestUrl": "/v1.0/teams/{team-id}/members",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/team-list-members?view=graph-rest-1.0&tabs=http",
+        "tip": "This query requires a team id. To find the team id of teams you belong to, you can switch to user (delegated permissions) mode and run: GET https://graph.microsoft.com/v1.0/me/joinedTeams.",
+        "skipTest": false
+    },
+    {
+        "id": "fe15123a-507a-45d8-aa29-e5b25f5c0ac7",
+        "category": "Microsoft Teams",
```
Comment (ayubun) [inline sample-queries/sample-queries.json]:
```suggestion
        "category": "Microsoft Teams (beta)",
```

File: sample-queries/sample-queries.json [hunk 2]
```diff
@@ -2984,44 +2984,54 @@
...
+        "tip": "This query requires a chat id. To find the chat ids, you can switch to user (delegated permissions) mode and run: GET https://graph.microsoft.com/beta/me/chats/",
+        "skipTest": false
+    },
+    {
+        "id": "da59a1e1-4979-485e-a128-44f28d6d67ac",
+        "category": "Microsoft Teams",
+        "method": "GET",
+        "humanName": "get list of members in team",
+        "requestUrl": "/v1.0/teams/{team-id}/members",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/team-list-members?view=graph-rest-1.0&tabs=http",
```
Comment (ayubun) [inline sample-queries/sample-queries.json:3022]:
```suggestion
        "category": "Microsoft Teams (beta)",
        "method": "GET",
        "humanName": "get list of members in team",
        "requestUrl": "/beta/teams/{team-id}/members",
        "docLink": "https://docs.microsoft.com/en-us/graph/api/team-list-members?view=graph-rest-beta",
```

File: sample-queries/sample-queries.json [hunk 3]
```diff
@@ -2984,44 +2984,54 @@
...
-            "docLink": "https://docs.microsoft.com/en-us/graph/api/channel-list-messages?view=graph-rest-beta&tabs=http",
-            "tip": "This query requires a team id and a channel id. To find the team id of teams you belong to, you can switch to user (delegated permissions) mode and run: GET https://graph.microsoft.com/v1.0/me/joinedTeams. To find the channel id of channels you belong to, stay in user (delegated permissions) mode and run: GET https://graph.microsoft.com/beta/teams/{team-id}/channels.",
-            "skipTest": false
-        }
+        "id": "44a072cc-c5c6-40f2-ab21-e7b0c417bb00",
+        "category": "Microsoft Teams",
+        "method": "GET",
+        "humanName": "channels of a team which I am member of",
+        "requestUrl": "/v1.0/teams/{team-id}/channels",
+        "docLink": "https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/channel_list",
```
Comment (ayubun) [inline sample-queries/sample-queries.json]:
```suggestion
        "category": "Microsoft Teams (beta)",
        "method": "GET",
        "humanName": "channels of a team which I am member of",
        "requestUrl": "/beta/teams/{team-id}/channels",
        "docLink": "https://docs.microsoft.com/en-us/graph/api/channel-list?view=graph-rest-beta",
```

**Review by ayubun — CHANGES_REQUESTED** [review]
I think for consistency, we should keep everything as beta and make sure the category labels correspond to such (otherwise they appear in different sections on the sample queries tab).

I know that two of these suggestions are not the queries that you added, but since you edited the indentation of the entire list so far, I thought it might as well be a good idea to add the changes here ^^

**Review by ayubun — APPROVED** [review]

**Pull request merged**